Corona-Warn-App/src/main/res/values-de/srs_submission_strings.xml

```diff
@@ -0,0 +1,119 @@
+    <string name="srs_submission_consent_first_bulletpoint_text">Ihre Identität bleibt geheim. Andere Nutzer erfahren nicht, wer sein Testergebnis geteilt hat.</string>
+    <!-- XTXT: Submission consent second bulletpoint -->
+    <string name="srs_submission_consent_second_bulletpoint_text">Unter „Meine Check-ins" können Sie Ihre Events und Orte einsehen, deren eingecheckte Gäste gewarnt werden. Sie können einzelne Check-ins auch löschen und so von der Warnung ausschließen.</string>
+    <!-- XTXT: Submission consent third bulletpoint -->
+    <string name="srs_submission_consent_third_bulletpoint_text">Sie können Ihr Einverständnis abgeben, wenn Sie mindestens 16 Jahre alt sind.</string>
+    <!-- XTXT: Submission consent next step submission -->
+    <string name="srs_next_step_submission_text">Erteilen Sie im nächsten Schritt Ihre Erlaubnis zum Zugriff auf die Zufalls-IDs.</string>
+    <!-- XTXT: Contact journal submission entry title -->
+    <string name="srs_contact_diary_submission_item_title">Andere gewarnt</string>
+
+    <!-- Type Selection Screen -->
+    <!-- XHED: Type selection header -->
+    <string name="srs_type_selection_header">"Art des Tests"</string>
+    <!-- XTXT: Type selection introduction -->
+    <string name="srs_type_selection_introduction_text">Bitte wählen Sie die Art des Tests aus, auf dessen Grundlage Sie warnen.</string>
+    <!-- XTXT: Type selection button rat registered no result text -->
+    <string name="srs_rat_registered_no_result_text">Antigen-Schnelltest in der App registriert, aber kein Ergebnis erhalten</string>
+    <!-- XTXT: Type selection button rat not registered text -->
+    <string name="srs_rat_not_registered_text">Antigen-Schnelltest nicht in der App registriert</string>
+    <!-- XTXT: Type selection button pcr registered no result text -->
+    <string name="srs_pcr_registered_no_result_text">PCR-Labortest in der App registriert, aber kein Ergebnis erhalten</string>
+    <!-- XTXT: Type selection button pcr not registered text -->
+    <string name="srs_pcr_not_registered_text">PCR-Labortest nicht in der App registriert</string>
+    <!-- XTXT: Type selection button rapid pcr text -->
+    <string name="srs_rapid_pcr_text">PCR-Schnelltest (PoC-NAT-Test)</string>
+    <!-- XTXT: Type selection button other text -->
+    <string name="srs_other_text">Sonstige / keine Angabe</string>
+    <!-- XTXT: Type selection next button text -->
+    <string name="type_selection_next_button_text">Weiter</string>
+
+    <!-- Srs Submission Cancel Dialog -->
+    <!-- XHED: Cancel submission dialog title -->
+    <string name="srs_cancel_dialog_title">Warn-Vorgang abbrechen?</string>
+    <!-- XTXT: Cancel submission dialog message -->
+    <string name="srs_cancel_dialog_message">Sind Sie sich sicher, dass Sie den Warn-Vorgang abbrechen wollen?\n\Wenn Ihr Test positiv war, können Sie mit einer Warnung helfen, Infektionsketten zu unterbrechen.</string>
```

Comment:
Is `\n\Wenn` correct?
Previously it was `\n\nWenn` and it looks like one `\n` was intended to be deleted.

Same issue in file `values` directory as well.

Reply:
https://github.com/corona-warn-app/cwa-app-android/pull/5709